context/AuthHelper.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { account } from '@/api/config';
+import Alert from '@/components/AlertNotification/AlertNotification';
+import { AlertVariants } from '@/components/AlertNotification/Alerts.enum';
+import { toast } from 'react-hot-toast';
+import { NextRouter } from 'next/router';
+import { getCurrentUser } from '@/api/apiFunctions';
+import { IUser } from '@/api/apiFunctions.interface';
+
+type UserCredentials = {
+  email: string;
+  password: string;
+};
+
+/**
+ * Authenticate and set session state
```

Comment:
Clean this up. There should only be 3 properties documented.

Reply:
I keep deleting the unnecessary properties, but every time it saves it auto adds in more.